src/back/spv/mod.rs

```diff
@@ -153,6 +153,70 @@ impl Function {
     }
 }
 
+fn map_dim(dim: crate::ImageDimension) -> spirv::Dim {
```

Comment:
any reason to not `impl From<>`?

Reply:
This was pre-existing. I'd much prefer a `From` impl. Done.